pkg/collector/corechecks/sbom/check.go

```diff
@@ -115,12 +115,12 @@ type Check struct {
 // Factory returns a new check factory
 func Factory(store workloadmeta.Component) optional.Option[func() check.Check] {
 	return optional.NewOption(func() check.Check {
-		return &Check{
+		return core.NewLongRunningCheckWrapper(&Check{
```

Comment:
can we use the same wrapper for all or other long running check? 
`container_image` `container_lifecycle`? 
It can be done in another PR, But I wanted to validate that this wrapper is generic enough todo so.

Reply:
```
    container_image
    ---------------
      Instance ID: container_image [OK]
      Long Running Check: true
      Configuration Source: file:/etc/datadog-agent/conf.d/container_image.yaml
      Total Metric Samples: 0
      Total Events: 0
      Total container-images: 147
      Total Service Checks: 0


    container_lifecycle
    -------------------
      Instance ID: container_lifecycle [OK]
      Long Running Check: true
      Configuration Source: file:/etc/datadog-agent/conf.d/container_lifecycle.d/conf.yaml.default
      Total Metric Samples: 0
      Total Events: 0
      Total container-lifecycle: 5
      Total Service Checks: 0
```
I guess it should work fine 👍